heron/tmaster/src/cpp/manager/tmaster.h

```diff
@@ -127,6 +134,15 @@ class TMaster {
   // Function called after we get the topology
   void GetTopologyDone(proto::system::StatusCode _code);
 
+  // Function called after we get StatefulConsistentCheckpoints
+  void GetStatefulCheckpointDone(proto::ckptmgr::StatefulConsistentCheckpoints* _ckpt,
```

Comment:
s/GetStatefulCheckpointDone/GetStatefulCheckpointsDone

And the same for the following methods.